src/chttpd_cors.erl

```diff
@@ -360,7 +399,7 @@ get_origin(Req) ->
         undefined ->
             undefined;
         Origin ->
-            ?l2b(to_lower(Origin))
+            ?l2b(Origin)
```

Comment:
Hmm not sure then. It contradicts [6.1 point 2](https://www.w3.org/TR/cors/#origin-request-header) where spec says:
> If the value of the Origin header is not a case-sensitive match for any of the values in list of origins, do not set any additional headers and terminate this set of steps.

I'll do some research on it.